package.json

```diff
@@ -133,16 +143,20 @@
     "webpack-cli": "^4.6.0"
   },
   "scripts": {
-    "start": "RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy yarn _private:start",
-    "start:dev": "RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy yarn _private:start:dev",
-    "start:test": "RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy RADICLE_UPSTREAM_PROXY_ARGS='--test' yarn _private:start",
+    "start": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy yarn _private:start",
+    "start:win": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy.exe yarn _private:start",
+    "start:dev": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy yarn _private:start:dev",
+    "start:dev:win": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy.exe yarn _private:start:dev",
+    "start:test": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy RADICLE_UPSTREAM_PROXY_ARGS='--test' yarn _private:start",
+    "start:test:win": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy.exe RADICLE_UPSTREAM_PROXY_ARGS='--test' yarn _private:start",
     "ethereum:start": "./scripts/ethereum-dev-node.sh",
-    "test": "TZ='UTC' yarn test:unit && TZ='UTC' yarn test:integration",
-    "test:integration": "TZ='UTC' run-p --race _private:proxy:start:test _private:test:integration",
-    "test:integration:debug": "TZ='UTC' run-p --race _private:webpack:ui:watch _private:proxy:start:test:watch _private:test:integration:debug",
+    "test": "cross-env TZ='UTC' yarn test:unit && TZ='UTC' yarn test:integration",
```

Comment:
I’m curious why we don’t use `cross-env` for the second command `TZ=UTC yarn test:integration`.

Reply:
I missed it hehe

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/2375.